client/public/site.webmanifest

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Is this a new file?

Reply:
Yeah, it was included in the guide I looked at and seems to allow the icon to show up if the user adds the webpage to their homescreen or something similar. [MDN Docs](https://developer.mozilla.org/en-US/docs/Web/Manifest).

Reply:
But it's not referenced anywhere...Are you adding this file because the change to `manifest.json` is not enough to get the icon changing?